src/main/java/com/kodilla/ecommercee/domain/dto/UserDto.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+public class UserDto {
+    private Long userId;
+    private String username;
+    private String status;
+    private int userKey;
+    private int cartId;
```

Comment:
W specu json są: id, username, status i userKey, więc pozostałe pola, raczej nie muszą być w DTO

Reply:
To jak w końcu robimy? bo widzę, że inni tez mają wszystkie pola z diagramu

Reply:
myślę że Dto powinny być zrobione wg specu json, a nie wg diagramu, ale w razie czego zawsze można to poprawić.